addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1129,6 +1129,42 @@ registry.FooterSlideout = publicWidget.Widget.extend({
     },
 });
 
+registry.TopMenuCollapse = publicWidget.Widget.extend({
+    selector: 'header #top_menu_collapse',
+
+    /**
+     * @override
+     */
+    async start() {
+        this.throttledResize = _.throttle(() => this._onResize(), 25);
+        window.addEventListener('resize', this.throttledResize);
+        return this._super(...arguments);
+    },
+    /**
+     * @override
+     */
+    destroy() {
+        this._super(...arguments);
+        window.removeEventListener('resize', this.throttledResize);
+    },
+
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     */
+    _onResize() {
+        if (this.el.classList.contains('show')) {
+            const togglerEl = this.el.closest('nav').querySelector('.navbar-toggler');
+            if (getComputedStyle(togglerEl).display === 'none') {
+                this.el.classList.remove('show');
```

Comment:
Good job, Benoit! I hope that fixing that this way will not be considered as overkill. IMO it's better/ more robust now :+1: 
Just I would not remove the `show` class that way. It's better to ask BS to hide the collapse. We never know.. there could be a `hidden.bs.collapse` event listener of something like that. It's always better to let Bootstrap handle its components :+1: